website/content/docs/v0.7/Learn More/talosctl.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "talosctl"
+weight: 7
+---
+
+The `talosctl` tool packs a lot of power into a small package.  It acts as a
+reference implementation for the Talos API, but it also handles a lot of
+conveniences for the use of Talos and its clusters.
+
+## Client Configuration
+
+Talosctl configuration is located in `$XDG_CONFIG_HOME/talos/config.yaml` if
+`$XDG_CONFIG_HOME` is defined.  Otherwise it is in `$HOME/.talos/config`.  The
+location can always be overridden by the `TALOSCONFIG` environment variable or the
+`--talosconfig` parameter.
+
+Like `kubectl`, `talosctl` uses the concept of configuration contexts, so any
+number of Talos clusters can be managed with a single configuration file.
+Unlike `kubectl`, it also comes with some intelligent tooling (inspired by Joey
+Espinosa's excellent `kconf` tool) to manage the merging of new contexts into
+the config.  The default operation is a non-destructive merge, where if a
+context of the same name already exists in the file, the context to be added is
+renamed by appending an index number.  You can easily overwrite instead, as
+well.  See the `talosctl config help` for more information.
+
+## Endpoints and Nodes
+
+![Endpoints and Nodes](/images/endpoints-and-nodes.png)
+
+Endpoints are the communication endpoints to which the client directly talks.
+These can be load balancers, DNS hostnames, a list of IPs, etc.
+In general, it is recommended that these point to the set of control plane
+nodes, either directly or through a reverse proxy or load balancer.
+
+Each endpoint will automatically proxy requests destined to another node through
+it, so it is not necessary to change the endpoint configuration just because you
+wish to talk to a different node within the cluster.
+
+Endpoints _do_, however, need to be members of the same Talos cluster as the
+target node, because these proxied connections reply on certificate-based
+authentication.
+
+The Node is the target node on which you wish to perform the API call.
+While you can configure the target node (or even set of target nodes) inside the
+'talosctl' configuration file, it is often useful to simply and explicitly
+declare the target node(s) using the `-n` or `--nodes` command-line parameter.
+
+Keep in mind, when specifying nodes that their IPs and/or hostnames are as seen by the endpoint servers, not as from the client.
+This is because all connections are proxied first through the endpoints.
+
+## Kubeconfig
+
+The configuration for accessing a Talos Kubernetes cluster is obtained with `talosctl`.
+By default, `talosctl` will use the [kconf](https://github.com/particledecay/kconf) package (built-in to `talosctl`) to safely merge the cluster into the default kubeconfig.
```

Comment:
I don't know if this is true anymore. @smira did we stop using that?

Reply:
Must have; it's not in the `go.mod`

Reply:
yep, we're using our own implementation now